kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -283,8 +287,9 @@ def output_size(self, detail=False):
             The maximum log-scale of the scattering transform. In other words,
             the maximum scale is given by `2 ** J`.
         {param_shape}Q : int
-            The number of first-order wavelets per octave (second-order
```

Comment:
Needs to be fixed to align with docstring above.

Reply:
Same align as original, no?

Reply:
This one talks about Q, Q2, while the above docstring talks about tuples.

Reply:
ah _that_ 'align'